docs/api/content.md

```diff
@@ -95,6 +95,29 @@ import CSSProps from '@site/static/usage/v7/content/theming/css-properties/index
 
 <CSSProps />
 
+### Safe Area Padding
+
+The content component will not automatically apply padding to any of its sides to account for the [safe area](/docs/layout/safe-area). This is because the content component is often used in conjunction with other components that apply their own padding, such as [headers](./header) and [footers](./footer). However, if the content component is being used on its own, it may be desired to apply padding to the safe area. This can be done through CSS by using the `--ion-safe-area-(dir)` variables described in [Application Variables](../theming/advanced.md#application-variables).
```

Comment:
this link and the notch one in line 114 aren't working in vercel

Reply:
Good catch, thank you!